sql/updates/characters/3.3.5/9999_99_99_99_characters.sql

```diff
@@ -0,0 +1,2 @@
+--
+DELETE FROM `pet_aura` WHERE `spell` IN (28801, 37025, 42201, 57634);
```

Comment:
One last thing before merge: the sql (and the characters_database.sql changes) can be dropped.

The core will automatically avoid loading any aura (and pet aura) which doesn't pass Aura::CanBeSaved

Reply:
Ready